package.json

```diff
@@ -12,14 +12,20 @@
   },
   "dependencies": {
     "@nextui-org/progress": "^2.0.31",
+    "@reduxjs/toolkit": "^2.2.7",
+    "he": "^1.2.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-redux": "^9.1.2",
     "react-router-dom": "^6.25.1",
-    "react-select": "^5.8.0"
+    "react-select": "^5.8.0",
+    "uuid": "^10.0.0"
```

Comment:
<img width="173" alt="Zrzut ekranu 2024-08-14 o 09 03 43" src="https://github.com/user-attachments/assets/6a911265-7622-49d2-85d7-295709802252">
I think this is a huge package to work with ids